src/content/protocols/kodiak-v3.md

```diff
@@ -0,0 +1,230 @@
+| MixedRouteQuoterV1  | [0xfa0276F06161cC2f66Aa51f3500484EdF8Fc94bB](https://berascan.com/address/0xfa0276F06161cC2f66Aa51f3500484EdF8Fc94bB) |
+| TickLens            | [0xa73C6F1FeC76D5487dC30bdB8f11d1F390394b48](https://berascan.com/address/0xa73C6F1FeC76D5487dC30bdB8f11d1F390394b48) |
+| KodiakIslandFactory | [0x5261c5A5f08818c08Ed0Eb036d9575bA1E02c1d6](https://berascan.com/address/0x5261c5A5f08818c08Ed0Eb036d9575bA1E02c1d6) |
+| IslandRouter        | [0x679a7C63FC83b6A4D9C1F931891d705483d4791F](https://berascan.com/address/0x679a7C63FC83b6A4D9C1F931891d705483d4791F) |
+| KodiakIsland        | [0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590](https://berascan.com/address/0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590) |
+| FarmFactory         | [0xAeAa563d9110f833FA3fb1FF9a35DFBa11B0c9cF](https://berascan.com/address/0xAeAa563d9110f833FA3fb1FF9a35DFBa11B0c9cF) |
+| KodiakFarm          | [0xEB81a9EEAF156d4Cfec2AF364aF36Ad65cF9f0fa](https://berascan.com/address/0xEB81a9EEAF156d4Cfec2AF364aF36Ad65cF9f0fa) |
+| PandaFactory        | [0xac335fe675699b0ce4c927bdaa572eb647ed9f02](https://berascan.com/address/0xac335fe675699b0ce4c927bdaa572eb647ed9f02) |
+
+## Permission owners
+
+| Name          | Account                                                                                                               | Type         |
+| ------------- | --------------------------------------------------------------------------------------------------------------------- | ------------ |
+| Safe Wallet 1 | [0x21802b7C3DF57e98df45f1547b0F1a72F2CD1aED](https://berascan.com/address/0x21802b7C3DF57e98df45f1547b0F1a72F2CD1aED) | Multisig 2/4 |
+| Safe Wallet 2 | [0x7D8A57bf453ecf440093fe6484c78B91b377e638](https://berascan.com/address/0x7D8A57bf453ecf440093fe6484c78B91b377e638) | Multisig 3/6 |
+| Safe Wallet 3 | [0x8A98f7A3dF97029bd8992966044a06DEe4CC5299](https://berascan.com/address/0x8A98f7A3dF97029bd8992966044a06DEe4CC5299) | Multisig 2/3 |
+
+## Permissions
+| Contract            | Function                   | Impact                                                                                                                               | Owner         |
+| ------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| UniswapV3Factory    | setOwner                   | Changes the owner address. Owner can modify fee protocol and amounts. Malicious owner could set extreme fees or disable fee amounts. | Safe Wallet 1 |
+| UniswapV3Factory    | setDefaultFeeProtocol      | Sets protocol fee percentage. Currently 35%. Malicious owner could set extreme fees, affecting all pool economics.                   | Safe Wallet 1 |
+| UniswapV3Factory    | enableFeeAmount            | Enables/disables fee tiers. Malicious owner could disable all fee tiers, breaking pool functionality.                                | Safe Wallet 1 |
```

Comment:
it does not break pool functionality, if all tiers removed, new pools cannot be deployed, existing pools are not affected (no loss of funds)